pkg/webhook/admission/pod/storage_initializer_injector.go

```diff
@@ -31,6 +33,7 @@ const (
 	StorageInitializerVolumeName            = "kfserving-provision-location"
 	StorageInitializerContainerImage        = "gcr.io/kfserving/storage-initializer"
 	StorageInitializerContainerImageVersion = "latest"
+	LocalURIPrefix                          = "file://"
```

Comment:
I'm not sure the controller should support file://. Is this something like a minikube scenario where the model and the pod are local?

Reply:
> I'm not sure the controller should support file://. Is this something like a minikube scenario where the model and the pod are local?

The file:// is already in controller and originally I want to remove it but it's recommended to keep it for local development. So here the fix is to skip the storage initializer if "file://" is specified for storageUri since it does not work. Need not download it in init container since the model is in the same context as the kfserver.